ChangeLog

```diff
@@ -21,6 +21,10 @@ Changes
      produced by some optimizing compilers, showing up as failures in
      e.g. RSA or ECC signature operations. Reported in #1722, fix suggested
      by Aurelien Jarno and submitted by Jeffrey Martin.
+   * Remove dead code from bignum.c in the default configuration.
+     Found by Coverity, reported and fixed by Peter Kolbus. Fixes #2309.
```

Comment:
Please credit my employer as well for both of these: "Peter Kolbus (Garmin)"

Reply:
Done. Thanks for a quick response!